Mathlib/Analysis/InnerProductSpace/JointEigenspace.lean

```diff
@@ -108,6 +105,65 @@ theorem directSum_isInteral_of_commute (hA : A.IsSymmetric) (hB : B.IsSymmetric)
 
 end Pair
 
+section Tuple
+
+universe u
+
+variable {n m : Type u}
```

Comment:
There's no reason to require that `n m` are in the *same* universe. Also, can you please put these at the top of the file with `𝕜 E`?
```suggestion
variable {n m : Type*}
```